romancal/resample/resample.py

```diff
@@ -732,30 +734,40 @@ def gwcs_into_l3(model, wcsinfo):
     by indexing. This is fragile and will be a source of issues.
     """
     l3_wcsinfo = model.meta.wcsinfo
-    transform = wcsinfo.forward_transform
+    transform = wcs.forward_transform
 
     # Basic WCS info
     l3_wcsinfo.projection = "TAN"
+    l3_wcsinfo.pixel_shape = model.shape
     l3_wcsinfo.rotation_matrix = transform["pc_rotation_matrix"].matrix.value.tolist()
     l3_wcsinfo.dec_ref = transform.lat_6.value
     l3_wcsinfo.ra_ref = transform.lon_6.value
-    # l3_wcsinfo.x_ref = center of mosaic?
-    # l3_wcsinfo.y_ref = center of mosaic?
-    # l3_wcsinfo.s_region = from gwcs.bounding_box
-
-    # Mosaic info
-    # l3_wcsinfo.dec_center = from center of mosaic
-    # l3_wcsinfo.pixel_scale = ???
-    # l3_wcsinfo.pixel_scale_local = ???
-    # l3_wcsinfo.pixel_shape = image shape
-    # l3_wcsinfo.ra_center = from center of mosaic
-    # l3_wcsinfo.ra_corn1 = from bounding box
-    # l3_wcsinfo.ra_corn2 = from bounding box
-    # l3_wcsinfo.ra_corn3 = from bounding box
-    # l3_wcsinfo.ra_corn4 = from bounding box
-    # l3_wcsinfo.dec_corn1 = from bounding box
-    # l3_wcsinfo.dec_corn2 = from bounding box
-    # l3_wcsinfo.dec_corn3 = from bounding box
-    # l3_wcsinfo.dec_corn4 = from bounding box
-    # l3_wcsinfo.orientat = ???
-    # l3_wcsinfo.orientat_local = ???
+    l3_wcsinfo.x_ref = -transform.offset_0.value  # cdelt1
+    l3_wcsinfo.y_ref = -transform.offset_1.value  # cdelt2
```

Comment:
The comment is wrong - it's crpix in FITS .
The current name of the models is `crpix1` and so `wcs.forward_transform['crpix1'].offset.value` is what is a better call. I don't like the current names and since I doubt they are used I may propose changing them in gwcs to `x_ref` and `y_ref`.